### DISCUSSION TITLE
Efficiency of the mechanical exhaust system

### DISCUSSION THREAD

**Discussion by quantumfds** [firemodels/fds#13644] [Q&A]
Hello,

In order to test our cluster, we made a quick test. There is a big storage area (80 x 60 m) with a fire in the middle. The HRR curve is "capped" when the sprinklers are activated with a delay of 30 s (temperature of 68 degrees Celsius). The mechanical exhaust system with 100 k m3/h is activated with smoke detectors at the ceiling (with 60 s for ramp-up time). The "sprinklers" are activated after around 3 minutes, and the smoke detectors in less than a minute.
We are mostly interested in the value of extinction coefficient at a height of 2 m. Currently, after 6 minutes at 2 m the visibility is under 20 m (using formula in FDS User Guide). With a fire of 1 MW and relatively big storage area, it seems that the smoke generation is very huge. Is there a way to double-check the results, because we are how sure if the mechanical exhaust system is working properly or did we actually made a mistake while writing the code.

Thanks.

Here is the code we used:

&HEAD CHID='test'/
&TIME T_END=1800.0/
&DUMP DT_RESTART=60.0, DT_SL3D=0.25/

&MESH ID='MESH-01', IJK=80,69,15, XB=0.0,32.0,-82.0,-54.4,0.0,6.0/
&MESH ID='MESH-02', IJK=80,68,15, XB=0.0,32.0,-54.4,-27.2,0.0,6.0/
&MESH ID='MESH-03', IJK=80,69,15, XB=0.0,32.0,-27.2,0.4,0.0,6.0/

&SPEC ID='SFPE POLYURETHANE_GM37_fuel', FORMULA='C1.0H1.2O0.2N0.08'/

&REAC ID='SFPE POLYURETHANE_GM37',
      FYI='SFPE Handbook, 5th Edition, Tables A.38 and A.39',
      FUEL='SFPE POLYURETHANE_GM37_fuel',
      CO_YIELD=0.024,
      SOOT_YIELD=0.113,
      HEAT_OF_COMBUSTION=1.79E+4,
      RADIATIVE_FRACTION=0.514/

&PROP ID='Cleary Photoelectric P1',
      QUANTITY='CHAMBER OBSCURATION',
      ALPHA_E=1.8,
      BETA_E=-1.0,
      ALPHA_C=1.0,
      BETA_C=-0.8/
&CTRL ID='FREEZE_TIME', FUNCTION_TYPE='TIME_DELAY', DELAY=30.0, LATCH=.FALSE., INPUT_ID='or'/
&CTRL ID='or', FUNCTION_TYPE='ANY', INPUT_ID='T1','T2','T3','T4'/
&CTRL ID='Exhaust_Activation', FUNCTION_TYPE='ANY', INPUT_ID='SD1','SD2','SD3','SD4'/
&DEVC ID='T1', QUANTITY='TEMPERATURE', XYZ=14.4,-38.8,5.2, SETPOINT=68.0/
&DEVC ID='T2', QUANTITY='TEMPERATURE', XYZ=18.4,-38.8,5.2, SETPOINT=68.0/
&DEVC ID='T3', QUANTITY='TEMPERATURE', XYZ=18.4,-42.4,5.2, SETPOINT=68.0/
&DEVC ID='T4', QUANTITY='TEMPERATURE', XYZ=14.4,-42.4,5.2, SETPOINT=68.0/
&DEVC ID='Time', QUANTITY='TIME', XYZ=0.0,0.0,0.0, NO_UPDATE_CTRL_ID='FREEZE_TIME'/
&DEVC ID='SD1', PROP_ID='Cleary Photoelectric P1', XYZ=14.0,-35.8,5.2/
&DEVC ID='SD2', PROP_ID='Cleary Photoelectric P1', XYZ=19.0,-35.8,5.2/
&DEVC ID='SD3', PROP_ID='Cleary Photoelectric P1', XYZ=14.0,-45.8,5.2/
&DEVC ID='SD4', PROP_ID='Cleary Photoelectric P1', XYZ=19.0,-45.8,5.2/
&DEVC ID='FLOW', QUANTITY='V-VELOCITY', SPATIAL_STATISTIC='AREA INTEGRAL', XB=12.8,19.2,0.0,0.0,4.4,5.6/

&SURF ID='Concrete',
      RGB=255,203,101,
      TRANSPARENCY=0.498039/
&SURF ID='Fire',
      COLOR='RED',
      HRRPUA=2083.0,
      RAMP_Q='Fire_RAMP_Q',
      TMP_FRONT=300.0/
&RAMP ID='Fire_RAMP_Q', T=0.0, F=0.0, DEVC_ID='Time'/
&RAMP ID='Fire_RAMP_Q', T=10.0, F=0.0/
&RAMP ID='Fire_RAMP_Q', T=20.0, F=0.01/
&RAMP ID='Fire_RAMP_Q', T=30.0, F=0.01/
&RAMP ID='Fire_RAMP_Q', T=40.0, F=0.03/
&RAMP ID='Fire_RAMP_Q', T=50.0, F=0.04/
&RAMP ID='Fire_RAMP_Q', T=60.0, F=0.06/
&RAMP ID='Fire_RAMP_Q', T=70.0, F=0.08/
&RAMP ID='Fire_RAMP_Q', T=80.0, F=0.1/
&RAMP ID='Fire_RAMP_Q', T=90.0, F=0.13/
&RAMP ID='Fire_RAMP_Q', T=100.0, F=0.16/
&RAMP ID='Fire_RAMP_Q', T=110.0, F=0.19/
&RAMP ID='Fire_RAMP_Q', T=120.0, F=0.23/
&RAMP ID='Fire_RAMP_Q', T=130.0, F=0.26/
&RAMP ID='Fire_RAMP_Q', T=140.0, F=0.31/
&RAMP ID='Fire_RAMP_Q', T=150.0, F=0.35/
&RAMP ID='Fire_RAMP_Q', T=160.0, F=0.4/
&RAMP ID='Fire_RAMP_Q', T=170.0, F=0.45/
&RAMP ID='Fire_RAMP_Q', T=180.0, F=0.51/
&RAMP ID='Fire_RAMP_Q', T=190.0, F=0.57/
&RAMP ID='Fire_RAMP_Q', T=200.0, F=0.63/
&RAMP ID='Fire_RAMP_Q', T=210.0, F=0.69/
&RAMP ID='Fire_RAMP_Q', T=220.0, F=0.76/
&RAMP ID='Fire_RAMP_Q', T=230.0, F=0.83/
&RAMP ID='Fire_RAMP_Q', T=240.0, F=0.9/
&RAMP ID='Fire_RAMP_Q', T=250.0, F=0.98/
&RAMP ID='Fire_RAMP_Q', T=300.0, F=1.0/
&RAMP ID='Fire_RAMP_Q', T=1800.0, F=1.0/
&SURF ID='Exhaust 100k',
      COLOR='YELLOW',
      HEAT_TRANSFER_COEFFICIENT=0.0,
      VOLUME_FLOW=27.77,
      TAU_V=-60.0/

&OBST ID='Obstruction', XB=7.017544E-3,32.0,-80.4,0.4,5.6,6.0, SURF_ID='Concrete'/ 
&OBST ID='Brand', XB=16.0,17.2,-41.2,-40.0,0.4,0.8, SURF_IDS='Fire','Concrete','Concrete'/ 
&OBST ID='Obstruction', XB=7.017544E-3,32.0,-80.4,0.4,0.0,0.4, SURF_ID='Concrete'/ 
&OBST ID='Obstruction', XB=0.0,0.4,-80.0,-54.4,0.0,5.6, SURF_ID='Concrete'/ 
&OBST ID='Obstruction', XB=0.0,32.0,-80.4,-80.0,0.0,5.6, SURF_ID='Concrete'/ 
&OBST ID='Obstruction', XB=31.6,32.0,-80.0,-54.4,0.0,5.6, SURF_ID='Concrete'/ 
&OBST ID='Obstruction', XB=0.0,0.4,-54.4,-27.2,0.0,5.6, SURF_ID='Concrete'/ 
&OBST ID='Obstruction', XB=31.6,32.0,-54.4,-27.2,0.0,5.6, SURF_ID='Concrete'/ 
&OBST ID='Obstruction', XB=0.0,0.4,-27.2,-9.670043E-14,0.0,5.6, SURF_ID='Concrete'/ 
&OBST ID='Obstruction', XB=0.4,32.0,-9.670043E-14,0.4,0.0,5.6, SURF_ID='Concrete'/ 
&OBST ID='Obstruction', XB=31.6,32.0,-27.2,-9.670043E-14,0.0,5.6, SURF_ID='Concrete'/ 
&OBST ID='Obstruction', XB=0.0,0.4,-9.670043E-14,0.4,0.0,0.0, SURF_ID='Concrete'/ 
&OBST ID='Obstruction', XB=0.0,0.4,-9.670043E-14,0.4,5.6,5.6, SURF_ID='Concrete'/ 
&OBST ID='Obstruction', XB=0.0,0.4,0.4,0.4,0.0,5.6, SURF_ID='Concrete'/ 
&OBST ID='Obstruction', XB=0.0,0.0,-9.670043E-14,0.4,0.0,5.6, SURF_ID='Concrete'/ 

&HOLE ID='Hole', XB=18.0,22.0,-80.8,-79.6,0.4,2.4/ 

&VENT ID='Mesh Vent: MESH-01 [XMAX]', SURF_ID='OPEN', XB=32.0,32.0,-82.0,-54.4,0.0,6.0/ 
&VENT ID='Mesh Vent: MESH-01 [XMIN]', SURF_ID='OPEN', XB=0.0,0.0,-82.0,-54.4,0.0,6.0/ 
&VENT ID='Mesh Vent: MESH-01 [YMIN]', SURF_ID='OPEN', XB=0.0,32.0,-82.0,-82.0,0.0,6.0/ 
&VENT ID='Mesh Vent: MESH-01 [ZMAX]', SURF_ID='OPEN', XB=0.0,32.0,-82.0,-54.4,6.0,6.0/ 
&VENT ID='Mesh Vent: MESH-01 [ZMIN]', SURF_ID='OPEN', XB=0.0,32.0,-82.0,-54.4,0.0,0.0/ 
&VENT ID='Mesh Vent: MESH-02 [XMAX]', SURF_ID='OPEN', XB=32.0,32.0,-54.4,-27.2,0.0,6.0/ 
&VENT ID='Mesh Vent: MESH-02 [XMIN]', SURF_ID='OPEN', XB=0.0,0.0,-54.4,-27.2,0.0,6.0/ 
&VENT ID='Mesh Vent: MESH-02 [ZMAX]', SURF_ID='OPEN', XB=0.0,32.0,-54.4,-27.2,6.0,6.0/ 
&VENT ID='Mesh Vent: MESH-02 [ZMIN]', SURF_ID='OPEN', XB=0.0,32.0,-54.4,-27.2,0.0,0.0/ 
&VENT ID='Mesh Vent: MESH-03 [XMAX]', SURF_ID='OPEN', XB=32.0,32.0,-27.2,0.4,0.0,6.0/ 
&VENT ID='Mesh Vent: MESH-03 [XMIN]', SURF_ID='OPEN', XB=0.0,0.0,-27.2,0.4,0.0,6.0/ 
&VENT ID='Mesh Vent: MESH-03 [YMAX]', SURF_ID='OPEN', XB=0.0,32.0,0.4,0.4,0.0,6.0/ 
&VENT ID='Mesh Vent: MESH-03 [ZMAX]', SURF_ID='OPEN', XB=0.0,32.0,-27.2,0.4,6.0,6.0/ 
&VENT ID='Mesh Vent: MESH-03 [ZMIN]', SURF_ID='OPEN', XB=0.0,32.0,-27.2,0.4,0.0,0.0/ 
&VENT ID='Exhaust 100k', SURF_ID='Exhaust 100k', XB=12.8,19.2,0.0,0.0,4.4,5.6, CTRL_ID='Exhaust_Activation'/ 

&ISOF QUANTITY='EXTINCTION COEFFICIENT', VALUE=0.2/
&ISOF QUANTITY='TEMPERATURE', VALUE=50.0/

&SLCF QUANTITY='VISIBILITY', ID='0', PBZ=2.4/
&SLCF QUANTITY='VELOCITY', ID='0', PBZ=2.4/
&SLCF QUANTITY='TEMPERATURE', ID='1', PBZ=2.4/
&SLCF QUANTITY='EXTINCTION COEFFICIENT', ID='2', PBZ=2.4/
&SLCF QUANTITY='TEMPERATURE', ID='3', PBY=-40.6/
&SLCF QUANTITY='TEMPERATURE', ID='4', PBX=16.8/
&SLCF QUANTITY='EXTINCTION COEFFICIENT', ID='4', PBX=16.8/
&SLCF QUANTITY='EXTINCTION COEFFICIENT', ID='3', PBY=-40.6/

&TAIL /

**Comment by drjfloyd** [comment]
You know the heat release rate of the fuel and the heat of combustion of the fuel. So you can convert kW of fire into kg/s of fuel.  You know the soot yield. So you can convert the kg/s of fuel into kg/s of soot.  You have the equation for visibility so you can convert your 2 m visibility into a soot density. You know the total volume of your domain so with a soot density you can determine the total kg of soot needed to give you 2 m visibility inside the volume. If you have kg/s of soot production and a total kg of soot needed you can determine at what time you would expect to see a visibility of 2 m. Obviously this will be an approximate time as this approach assumes uniform density everywhere.

**Comment by quantumfds** [comment]
Thanks, I am not sure how I can convert the visibility into a soot density. In the Users Guide, there is the equation related to the extinction coefficient, if that is what you meant.

**Reply by drjfloyd** [reply]
See 22.10.5 in the User's Guide.  The visiblity equation is there and all of its terms are defined.

**Reply by quantumfds** [reply]
I still do not see relationship between equations 22.21, 22.22 and soot density (at least not directly). The aim is to have visibility of at least 20 m at the height of 2 m. It just seems not usual that 1 MW fire generates so much smoke so that visibility drops very fast, although there is a mechanical exhaust system with 100 k m3/h in a big storage area of almost 24.000 m3 of volume.

**Reply by drjfloyd** [reply]
This is very basic fire dynamics and unit analysis

Eq 22.23 S = C / K
Eq 22.22 K = rho Y_S K_m
S = 2, C = 3, K_m = 8700 so rho Y_s = 0.000172 kg/m3

Your volume above 2 m is 80 x 60 x 4 = 19200 m3 which means to get 2 m of visibility you need 3.3 kg of soot.

Your fire size is 1 MW with a 17900 kJ/kg heat of combustion so your fuel flow rate is 0.0559 kg/s with a soot yield of 11.5 % so the soot production rate is 0.00631 kg/s.  At that rate is would take 524 s to fill the upper 4 m with 3.3 kg of soot with no exhaust running. 

Your exhaust rate is about 3.5 air changes per hour. In 524 s you have done about 50 % of an air change. 

You are certainly producing enough soot to get that level of visibility.
